app/src/shared/modules/user/actions.js

```diff
@@ -40,7 +40,7 @@ export const resetUserData: ReduxActionCreator = createAction(RESET_USER_DATA)
 export const logout = () => (dispatch: Function) => {
     clearStorage()
     dispatch(resetUserData())
-    dispatch(push(routes.logout()))
```

Comment:
@juhah throw an eye. Here we're trying to redirect to `/logout` while being there already. Changed to `/`.

Reply:
This is at least used after deleting an account, probably not elsewhere:

https://github.com/streamr-dev/streamr-platform/blob/cea513aac6e6423bfe0807e08bad18f25b1585b0/app/src/userpages/components/ProfilePage/DeleteAccount/index.jsx#L43

So that's a special case that we can't say that we are already in `/logout`. Does resetting user data and redirecting to `/` take care of that though?

Reply:
@juhah thanks, very helpful. I'll make sure it works for both scenarios.

Reply:
@juhah, I confirm that this new code also logs deleted account out.